app/screens/manage_channel_members/index.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+
+import {observeProfileLongPresTutorial} from '@app/queries/app/global';
+import {observeChannel} from '@queries/servers/channel';
+import {observeCurrentTeamId} from '@queries/servers/system';
+import {observeCurrentUser, observeTeammateNameDisplay} from '@queries/servers/user';
+
+import ManageChannelMembers from './manage_channel_members';
+
+import type {WithDatabaseArgs} from '@typings/database/database';
+
+type Props = WithDatabaseArgs & {
+    channelId: string;
+}
+
+const enhanced = withObservables(['channelId'], ({channelId, database}: Props) => {
+    return {
+        currentChannel: observeChannel(database, channelId),
+        currentUser: observeCurrentUser(database),
```

Comment:
from the current user you only need the id and to know if is admin, you don't need the full object